server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -595,6 +595,16 @@ public final class IndexSettings {
         Property.Dynamic
     );
 
+    public static final TimeValue DEFAULT_REMOTE_TRANSLOG_BUFFER_INTERVAL = new TimeValue(650, TimeUnit.MILLISECONDS);
+    public static final TimeValue MINIMUM_REMOTE_TRANSLOG_BUFFER_INTERVAL = TimeValue.ZERO;
+    public static final Setting<TimeValue> INDEX_REMOTE_TRANSLOG_BUFFER_INTERVAL_SETTING = Setting.timeSetting(
```

Comment:
As we don't have a validation here, is it possible to provide the setting without enabling remote translog?

Reply:
Yes, validation is removed to allow the settings to be dynamic. The current validation framework expects that the dependent setting should be passed also in the update payload. However, the dependent settings are final and it leads to update call failing all the time.